JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -16,11 +16,6 @@ module Microsoft.ApplicationInsights.Context {
         public id: string;
 
         /**
-         * The true if this is the first session
-         */
-        public isFirst: boolean;
```

Comment:
Yes, you are correct. We are still using it to detect "new users" vs. "returning users". I really wonder why it is not a field on SessionState event. But nevertheless I returned it back